R/env.R

```diff
@@ -70,7 +70,7 @@
 #'   features.
 #' @param .parent,parent A parent environment. Can be an object
 #'   supported by [as_environment()].
-#' @seealso `scoped_env`, [env_has()], [env_bind()].
```

Comment:
I think the link to `scoped_env` should be removed altogether as the API is uncertain.